Core/src/main/java/com/plotsquared/core/plot/Plot.java

```diff
@@ -1201,12 +1219,6 @@ public boolean removeFlag(final @NonNull PlotFlag<?, ?> flag) {
      *
      * @return array of entity counts
      * @see RegionManager#countEntities(Plot)
-     *         0 = Entity
-     *         1 = Animal
-     *         2 = Monster
-     *         3 = Mob
-     *         4 = Boat
-     *         5 = Misc
```

Comment:
^This isn't addressed? It would be better to link it

Reply:
That is copied from the javadocs of `RegionManager#countEntities(Plot)`, which is referenced.

Reply:
But the @see isn't clickable, and the PR seems to be making sure clickable @see stuff isn't in @see

Reply:
The PR moves additional documentation out of `@see`, because it doesn't belong here. `@see` acts like `{@link}`, just without the mandatory linking reference, because it expects a method/package/class/etc. only.

Reply:
Ah gotcha